doc/API-Mastodon.md

```diff
@@ -132,13 +231,24 @@ These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/en
 - [`GET /api/v1/tags/:id/unfollow`](https://docs.joinmastodon.org/methods/tags/#unfollow)
 - [`GET /api/v1/timelines/direct`](https://docs.joinmastodon.org/methods/timelines/)
 - [`GET /api/v1/timelines/home`](https://docs.joinmastodon.org/methods/timelines/)
+	- Additional field `with_muted` Pleroma extension to return notifications from muted users, defaults to `false`
+    - Additional field `exclude_replies` to only return post statuses not replies/comments, defaults to `false`
 - [`GET /api/v1/timelines/list/:id`](https://docs.joinmastodon.org/methods/timelines/)
+	- Additional field `with_muted` Pleroma extension to return notifications from muted users, defaults to `false`
+	- Additional field `exclude_replies` to only return post statuses not replies/comments, defaults to `false`
 - [`GET /api/v1/timelines/public`](https://docs.joinmastodon.org/methods/timelines/)
+	- Additional field `with_muted` Pleroma extension to return notifications from muted users, defaults to `false`
+	- Additional field `exclude_replies` to only return post statuses not replies/comments, defaults to `false`
 - [`GET /api/v1/timelines/tag/:hashtag`](https://docs.joinmastodon.org/methods/timelines/)
+	- Additional field `with_muted` Pleroma extension to return notifications from muted users, defaults to `false`
+	- Additional field `exclude_replies` to only return post statuses not replies/comments, defaults to `false`
+    - Does not support the `any[]`, `all[]`, or `none[]` query parameters
```

Comment:
Indentation is different from the previous lines.

Reply:
I've looked at this in two different editors (PHPSTorm and VSCodium) and this should be consistent now. If it's not I give up how to get it so on my computer.